stubs/pep8-naming/pep8ext_naming.pyi

```diff
@@ -13,22 +14,29 @@ METACLASS_BASES: Final[frozenset[Literal["type", "ABCMeta"]]]
 METHOD_CONTAINER_NODES: Final[set[ast.AST]]
 FUNC_NODES: Final[tuple[type[ast.FunctionDef], type[ast.AsyncFunctionDef]]]
 
-class _ASTCheckMeta(type):
-    codes: tuple[str, ...]
+class BaseASTCheck:
     all: list[BaseASTCheck]
-    def __init__(cls, class_name: str, bases: tuple[object, ...], namespace: Iterable[str]) -> None: ...
-
-class BaseASTCheck(metaclass=_ASTCheckMeta):
     codes: tuple[str, ...]
-    all: list[BaseASTCheck]
+    def __init_subclass__(cls, **kwargs: Any) -> None: ...
```

Comment:
```suggestion
    # Per convention, unknown kwargs are passed to the super-class. See there for the types.
    def __init_subclass__(cls, **kwargs: Any) -> None: ...
```

(Which are non-existent here, but maybe something like this could become a standard text for `__init_subclass__`.)